pkg/render/kubecontrollers/kube-controllers.go

```diff
@@ -528,16 +527,6 @@ func (c *kubeControllersComponent) controllersDeployment() *appsv1.Deployment {
 		VolumeMounts:    c.kubeControllersVolumeMounts(),
 	}
 
-	if c.kubeControllerName == EsKubeController {
```

Comment:
I am pretty sure es-calico-kube-controllers still talks to elasticsearch - we still use it in a few scenarios. To name a few:

- For Kibana configuration in any cases that Kibana is enabled.
- Support for managed clusters <= v3.16 (pre linseed)
- Installing the LInseed user in single-tenant clusters

Reply:
Both my standalone and single-tenant management-managed clusters seem ok without this and I can't find any references in existing kube-controllers code to either raw `ELASTIC_*` env vars or LMA calls to fetch/load them. I can revert it for the version skew situation alone though. I'll verify their usage in the versions we'll still have to support though, would 3.16 be the oldest we have to worry about? Or do we have to support even further back?

Reply:
Added back only the specific values we need to support the 3.16 managed cluster case.